WcaOnRails/app/assets/stylesheets/users.scss

```diff
@@ -20,6 +20,13 @@ td.unconfirmed-email {
   font-style: italic;
 }
 
+.pending-id {
+  background-color: #f2dede;
+  border-color: #ebccd1;
+  color: #a94442;
+  padding: 2px;
```

Comment:
Any reason for that? As far as I can imagine, this makes claimed and unclaimed WCA IDs look differently in the table.